ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -61,6 +61,11 @@ tests:
   steps:
     test:
     - ref: stackrox-e2e
+- as: gke-sensor-integration-tests
```

Comment:
You will need to add a `merge-` a `postsubmit: true` version of this.

Reply:
Added that in [fec3275](https://github.com/openshift/release/pull/40290/commits/fec32759fea1fdc1840a8e55a250542dc96f125c)